### PR TITLE
Basics scraping

### PR DESCRIPTION
**Pull request by ragog**

### CODE REVIEW

**Review by tnolet — CHANGES_REQUESTED** [review]
Good stuff, just one niggle

File: blog/snippets/playwright/basic-get-image.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const axios = require("axios");
+const fs = require("fs");
+
+(async () => {
+  const browser = await chromium.launch();
+  const page = await browser.newPage();
+  await page.goto("https://danube-webshop.herokuapp.com");
+  const url = await page.$eval("img", (el) => el.src);
+
+  axios
```
Comment (tnolet) [inline blog/snippets/playwright/basic-get-image.js:11]: We can use `await` syntax here to make it shorter. Also explain the use of `axios` in the main text.

**Review by tnolet — CHANGES_REQUESTED** [review]

File: blog/snippets/playwright/basic-get-image.js (hunk 1 above)
Comment (tnolet) [reply blog/snippets/playwright/basic-get-image.js:11]: sorry, my comment was terrible. The `axios` calls can also use `await`

Comment (ragog) [reply blog/snippets/playwright/basic-get-image.js:11]: Understood, makes total sense. Applying.

**Pull request merged**